### DISCUSSION TITLE
Refactor automatic setting of shipping method, `getShippingMethod()` and `shippingMethodHandle` on Order element

### DISCUSSION THREAD

**Discussion by nfourtythree** [craftcms/commerce#2387] [Ideas]
Potentially introduce a setting for automatically setting a shipping method on a cart.

**Comment by sparkalow** [comment]
Seconding.

A config setting would be much appreciated.  We have a few shipping needs that would benefit from this. (handling un-shippable items from a provider API, bring your own shipping account, a free pickup option along with paid options)
